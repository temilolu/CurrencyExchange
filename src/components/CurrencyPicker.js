import React from 'react';

const CurrencyPicker = ({
	currenyOptions,
	name,
	currencyValue,
	selectCurrency
}) => {
	return (
		<React.Fragment>
			<select
				className='custom-select custom-select-sm mb-3'
				name={name}
				onChange={selectCurrency}
				value={currencyValue}>
				{currenyOptions}
			</select>
		</React.Fragment>
	);
};

export default CurrencyPicker;
