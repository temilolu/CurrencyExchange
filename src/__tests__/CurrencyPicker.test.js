import React from 'react';
import CurrencyPicker from '../components/CurrencyPicker';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

afterEach(cleanup);

describe('<CurrencyPicker /> spec', () => {
	it('Should matches snapshot', () => {
		const { asFragment } = render(<CurrencyPicker />);
		expect(asFragment()).toMatchSnapshot();
	});
});
