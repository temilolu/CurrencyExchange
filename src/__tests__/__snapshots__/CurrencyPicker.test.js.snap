// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CurrencyPicker /> spec Should matches snapshot 1`] = `
<DocumentFragment>
  <select
    class="custom-select custom-select-sm mb-3"
  />
</DocumentFragment>
`;
